artemis-hawtio/artemis-plugin/src/main/webapp/plugin/js/brokerDiagram.js

```diff
@@ -455,16 +457,27 @@ var ARTEMIS = (function(ARTEMIS) {
                               addLinkIds("broker:" + broker.brokerId, "broker:" + "\"" + remoteBroker.live + "\"", "network");
 
                               var backup = remoteBroker.backup;
+                              ARTEMIS.log.info("isbackup? " + backup);
                               if (backup) {
                                  getOrAddBroker(false, "\"" + backup + "\"", remoteBroker.nodeID, "remote", null, properties);
                                  addLinkIds("broker:" + "\"" + remoteBroker.live + "\"", "broker:" + "\"" + backup + "\"", "network");
                               }
                            }
                            else {
-                              var backup = remoteBroker.backup;
-                              if (backup) {
-                                 getOrAddBroker(false, "\"" + remoteBroker.backup + "\"", remoteBroker.nodeID, "remote", null, properties);
-                                 addLinkIds("broker:" + broker.brokerId, "broker:" + "\"" + remoteBroker.backup + "\"", "network");
+                              var newBackReq = ARTEMISService.artemisConsole.isBackup(jolokia, mBean);
+                              var newBackup = newBackReq.value;
+                              if (!newBackup) {
+                                 ARTEMIS.log.info("yes I'm master right now");
+                                 if (remoteBroker.backup) {
+                                    getOrAddBroker(false, "\"" + remoteBroker.backup + "\"", remoteBroker.nodeID, "remote", null, properties);
+                                    addLinkIds("broker:" + broker.brokerId, "broker:" + "\"" + remoteBroker.backup + "\"", "network");
+                                 }
+                              }
+                              else {
+                                 ARTEMIS.log.info("ok I'm backup!");
```

Comment:
debug message

Reply:
Yes should be debug. I'll change that. Thx.